x-pack/plugins/security_solution/server/endpoint/lib/artifacts/saved_object_mappings.ts

```diff
@@ -34,3 +34,31 @@ export const manifestType: SavedObjectsType = {
   mappings: manifestSavedObjectMappings,
   migrations,
 };
+
+export const unifiedManifestSavedObjectType = ManifestConstants.UNIFIED_SAVED_OBJECT_TYPE;
+
+export const unifiedManifestSavedObjectMappings: SavedObjectsType['mappings'] = {
+  dynamic: false,
+  properties: {
+    artifactIds: {
+      type: 'text',
+    },
+    policyId: {
+      type: 'keyword',
+    },
+    semanticVersion: {
+      type: 'text',
+    },
+    created: {
+      type: 'date',
+    },
+  },
+};
+
+export const unifiedManifestType: SavedObjectsType = {
```

Comment:
Exported yet not imported anywhere - for now we do not want to commit the new SO type because there might be some adjustments needed during implementation of manifest manager.